### PR TITLE
Review on project from trials

### PR DESCRIPTION
**Pull request by io-oseinana**
# Review on project

In this repo I:

- Create a new repo
- Create a new branch for old project
- Copy my Admission trials project for review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      <div>
+        <input type="checkbox" value="AC" id="">
+        <label for="">Additional Courses</label>
+      </div>
+    </div>
+    <div>
+      <p>Any comments or suggestions?</p>
+      <textarea name="" id="" cols="50" rows="7" placeholder="Enter your comment here"></textarea>
+    </div>
+    <button id="submit" type="submit">Submit</button>
```
Comment (io-oseinana) [inline index.html:81]: Please I suggest you use classes in your code. It makes you code clean and understandable

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  width: 100%;
+  padding: 12px 20px;
+  border: none;
+  border-radius: 6px;
+  background-color: #4caf50;
+}
+
+button:hover {
+  background-color: #81e485;
+}
```
Comment (io-oseinana) [inline css/style.css:78]: Please you can use classes for `div` and `button`, Using Classes help you to avoid any bug in the future 😊

**Review by io-oseinana — COMMENTED** [review]
Hi! Great job so far

- [x] Correct Github Work flow
- [x] Nice description for PR


# Changes required :recycle: 

Please there's no linters to check for error in your code base and README file is not documented in professional way.

- [ ] No linters
- [ ] README.md is not documented Professionally
photo of readme.md



# Optional Changes

**Review by Nyame-Wolf — APPROVED** [review]
# STATUS: APPROVED :heavy_check_mark: :green_circle: 

Hi @io-oseinana  :smile: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :sparkles:

## Optional suggestions :ballot_box_with_check: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

APPROVED! 💯 💯 💯

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me, @Nyame-Wolf in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**